### PR TITLE
Ros2 lynx description

### PR DESCRIPTION
**Pull request by KmakD**
### Description

[Summary of the changes]

### Modifications

-

### CODE REVIEW

File: lynx_description/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+The package contains URDF files responsible for creating a representation of the robot by specifying the relationships and types of connections (joints) between individual links. It also contains information about the robot's mesh.
+
+## Launch Files
+
+- `load_urdf.launch.py` - loads the robot's URDF and creates simple bindings to display moving joints.
+
+## Configuration Files
+
+- [`components.yaml`](./config/components.yaml): Allows you to quickly add visualization of sensors, TF connections and simulate their behavior in the simulator.
+- [`WH05.yaml`](./config/WH04.yaml): Description of physical and visual parameters for the wheel WH05.
```
Comment (rafal-gorecki) [inline lynx_description/README.md]:
```suggestion
- [`WH05.yaml`](./config/WH05.yaml): Description of physical and visual parameters for the wheel WH05.
```

File: lynx_description/env-hooks/lynx_description.sh.in [hunk 2]
```diff
@@ -0,0 +1,3 @@
+ament_prepend_unique_value GAZEBO_MODEL_PATH "@CMAKE_INSTALL_PREFIX@/share"
```
Comment (rafal-gorecki) [inline lynx_description/env-hooks/lynx_description.sh.in]: Folder name will be changed on just `hooks` on the `ros-sim-estop`

File: lynx_description/config/components.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# By default panther is loaded without any components.
```
Comment (rafal-gorecki) [inline lynx_description/config/components.yaml]:
```suggestion
# By default lynx is loaded without any components.
```

File: lynx_description/urdf/wheel.urdf.xacro [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+        <inertia
+          ixx="${wheel_config['inertia']['ixx']}"
+          ixy="0.0"
+          ixz="0.0"
+          iyy="${wheel_config['inertia']['iyy']}"
+          iyz="0.0"
+          izz="${wheel_config['inertia']['izz']}" />
+      </inertial>
+    </link>
+
```
Comment (rafal-gorecki) [inline lynx_description/urdf/wheel.urdf.xacro:73]: If we add gazebo.urdf I propose to add also gazebo friction to not forget about it later.

```suggestion

    <xacro:if value="${use_sim}">
      <xacro:if value="${wheel_config['mecanum']}">
        <gazebo reference="${prefix}_wheel_link" xmlns:ignition="http://ignitionrobotics.org/schema">
          <collision>
            <surface>
              <friction>
                <ode>
                  <mu>1.0</mu>
                  <mu2>0.0</mu2>
                  <fdir1 ignition:expressed_in="base_link">${fdir}</fdir1>
                </ode>
              </friction>
            </surface>
          </collision>
        </gazebo>
      </xacro:if>
    </xacro:if>

```

File: panther_controller/config/WH05_controller.yaml [hunk 5]
```diff
@@ -0,0 +1,84 @@
+/**:
```
Comment (rafal-gorecki) [inline panther_controller/config/WH05_controller.yaml:1]: If there are plans to create a lynx controller, you can create it now. If you can resolve the comment.

**Review by rafal-gorecki — CHANGES_REQUESTED** [review]

File: panther_controller/config/WH05_controller.yaml (hunk 5 above)
Comment (KmakD) [reply panther_controller/config/WH05_controller.yaml:1]: The `WH05_controller.yaml` is intended to be a default lynx controller. WH05 most likely will be name of default lynx wheels. For now I placed it in panther_controller package, but later a dedicated package for lynx will be created.

**Review by rafal-gorecki — APPROVED** [review]

**Pull request merged**